diffs/packetbeatdiff.diff

```diff
@@ -0,0 +1,2602 @@
+-                <a id="_literal_metadata_literal">
++                <a id="_metadata">
+                 </a>
+                 <code class="literal">
+                  metadata
+@@ -891,7 +891,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_max_retries_literal_3">
++                <a id="_max_retries_3">
+                 </a>
+                 <code class="literal">
+                  max_retries
+@@ -922,7 +922,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_bulk_max_size_literal_3">
++                <a id="_bulk_max_size_3">
+                 </a>
+                 <code class="literal">
+                  bulk_max_size
+@@ -943,7 +943,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_bulk_flush_frequency_literal">
++                <a id="_bulk_flush_frequency">
+                 </a>
+                 <code class="literal">
+                  bulk_flush_frequency
+@@ -964,7 +964,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_timeout_literal_4">
++                <a id="_timeout_4">
+                 </a>
+                 <code class="literal">
+                  timeout
+@@ -985,7 +985,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_broker_timeout_literal">
++                <a id="_broker_timeout">
+                 </a>
+                 <code class="literal">
+                  broker_timeout
+@@ -1006,7 +1006,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_channel_buffer_size_literal">
++                <a id="_channel_buffer_size">
+                 </a>
+                 <code class="literal">
+                  channel_buffer_size
+@@ -1027,7 +1027,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_keep_alive_literal">
++                <a id="_keep_alive">
+                 </a>
+                 <code class="literal">
+                  keep_alive
+@@ -1048,7 +1048,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_compression_literal">
++                <a id="_compression">
+                 </a>
+                 <code class="literal">
+                  compression
+@@ -1089,7 +1089,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_compression_level_literal_3">
++                <a id="_compression_level_3">
+                 </a>
+                 <code class="literal">
+                  compression_level
+@@ -1141,7 +1141,7 @@
+              <div>
+               <div>
+                <
```

Comment:
Fixed in https://github.com/elastic/beats/pull/13722/commits/6c28fb10d52f74646c8868ed1c7981fcf85fedf4